### PR TITLE
Update get screen size from window to context #823462745446780

### PR DESCRIPTION
**Pull request by jessehon**
https://app.asana.com/0/683768070860536/823462745446780

### CODE REVIEW

File: app/javascript/App/components/ItemSelectorCategory.js [hunk 1]
```diff
@@ -32,50 +36,51 @@ export default class ItemSelectorCategory extends Component {
...
-                  <ItemButton item={itemLabel} style={{cursor:'pointer'}} />
-                </div>
-              )
-            })}
-          </div>);
+          return (
+            <div className="row" key={index}>
+              {itemGroups.map((item, innerIndex) => {
+                let itemLabel
+                if (item === 'Events - Conferences, Meetups, Launches, etc.') {
```
Comment (hanchang) [inline app/javascript/App/components/ItemSelectorCategory.js]: I think we can safely nuke these with the release of the new NewsCategories.

File: app/javascript/App/components/CalendarPage/CalendarList.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ class CalendarList extends Component {
```
Comment (hanchang) [inline app/javascript/App/components/CalendarPage/CalendarList.js]: Does this file also need `import withDevice` and a corresponding HOC wrapper in the `export default`?

**Comment by hanchang** [comment]
Looks like a great start!

File: app/javascript/App/components/CalendarPage/CalendarList.js (hunk 2 above)
Comment (jessehon) [reply app/javascript/App/components/CalendarPage/CalendarList.js]: Yes it does, missed that change

File: app/javascript/App/components/ItemSelectorCategory.js (hunk 1 above)
Comment (jessehon) [reply app/javascript/App/components/ItemSelectorCategory.js]: Still referenced in the `FilterPanel` by the `CalendarPage` it seems

Comment (hanchang) [reply app/javascript/App/components/ItemSelectorCategory.js]: Ah, thanks for the catch - we can safely ignore Calendar as it basically needs to be rebuilt sans Redux anyways.

**Pull request merged**